ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -47,17 +48,16 @@ class FaultInjector
     VelocityLimit=7, 
     NoForceData=8};
 
-  enum SystemFaults {
-    System=1, 
-    ArmGoalError=2, 
-    ArmExecutionError=4,
-    TaskGoalError=8, 
-    CamGoalError=16, 
-    CamExecutionError=32, 
-    PtGoalError=64, 
-    PtExecutionError=128, 
-    LanderExecutionError = 256};
-
+	static constexpr std::bitset<9> isSystem{		0b0'0000'0001 };
+	static constexpr std::bitset<9> isArmGoalError{		0b0'0000'0010 };
+	static constexpr std::bitset<9> isArmExecutionError{		0b0'0000'0100 };
+	static constexpr std::bitset<9> isTaskGoalError{	0b0'000'1000 };
+	static constexpr std::bitset<9> isCamGoalError{	0b0'0001'0000 };
+	static constexpr std::bitset<9> isCamExecutionError{	0b0'0010'0000 };
+	static constexpr std::bitset<9> isPtGoalError{		0b0'0100'0000 };
+	static constexpr std::bitset<9> isPtExecutionError{	0b0'1000'0000 };
+	static constexpr std::bitset<9> isLanderExecutionError{	0b1'0000'0000 };
```

Comment:
Lander seems pretty general - what exactly does this mean?  Also, I presume you plan to add power faults, under another ticket of course.

Reply:
Yea we can add Power Faults, it's not here since JPL never defined it for their system and we have slightly different lander systems. But I can make a new small PR to add that in for us!